src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -125,3 +131,10 @@ def _persist_run(self, run: Run) -> None:
             self._convert_run_to_spans(run.dict())
         except Exception:
             logger.exception("Failed to convert run to spans")
+
+    def get_spans(self) -> List[Span]:
+        """
+        Returns the spans stored in the tracer. This is useful if you are running
+        LangChain in a notebook environment and you want to inspect the spans.
+        """
+        return self.span_buffer
```

Comment:
Use iterator to prevent tempering (with the actual list in memory). Also, shouldn't this be in the `Tracer` base class?
```suggestion
    def get_spans(self) -> Iterator[Span]:
        """
        Returns the spans stored in the tracer. This is useful if you are running
        LangChain in a notebook environment and you want to inspect the spans.
        """
        yield from self.span_buffer
```

Reply:
Ah didn't know about the `Iterator` nuance. Yeah I can move it to the base class 👍